understanding/label-in-name.html

```diff
@@ -273,82 +190,61 @@ <h4>Mathematical expressions and formulae</h4>
+                     <li>コンポーネントのセットのグループラベル (すなわち、<code>legend</code>/<code>fieldset</code> で使用される、又は <code>group</code> もしくは <code>radiogroup</code> の役割 (role) で使用される)
                      </li> 
                      
                   </ul>
                   
-                  <p>Such textual information may constitute part of the component's <em>description</em>. So from both a programmatic viewpoint, and from the conservative tactic of only
-                     considering a label to be "adjacent text," neither headings, instructions, nor group
-                     'labels' should normally be considered <strong>labels</strong> for the purpose of this Success Criterion.
+                  <p>そのようなテキスト情報は、コンポーネントの<em>説明</em>の一部を構成する場合がある。したがって、プログラムの観点からと、ラベルを「隣接するテキスト」と見なすという保守的な戦術の両方から、通常、見出し、指示、グループの「ラベル」のいずれも、この達成基準の目的の<strong>ラベル</strong>と見なされるべきではない。
                   </p>
                   
-                  <p>It is important to note that the specification allows authors to override the name
-                     calculated through native semantics. Both <code>aria-label</code> and <code>aria-labelledby</code> take precedence in the name calculation, overriding the visible text as the accessible
-                     name even when the visible text label is programmatically associated with the control.
-                     For this reason, when a visible label already exists, <code>aria-label</code> should be avoided or used carefully, and <code>aria-labelledby</code> should be used as a supplement with care.
+                  <p>この仕様では、コンテンツ制作者がネイティブセマンティクスによって計算された名前 (name) を上書きできることに注意することが重要である。名前 (name) の計算では、<code>aria-label</code> と <code>aria-labelledby</code> の両方が優先され、表示されるテキストラベルがプログラムでコントロールに関連付けられている場合でも、アクセシブルな名前 (accessible name) として表示されるテキストが上書きされる。このため、可視ラベルがすでに存在する場合、<code>aria-label</code> を避ける又は慎重に使用すべきであり、<code>aria-labelledby</code> は注意して補足として使用すべきである。
                   </p>
                   
-                  <p>Finally, <code>aria-describedby</code> is not included in the Accessible Name computation (instead it is part of the Accessible
-                     Description computation). By convention, text associated with a control through <code>aria-describedby</code> is announced immediately after the accessible name by screen readers. Therefore,
-                     the context of headings, instructions, and group labels can be provided through the
-                     accessible description to assist users of screen readers without affecting the experience
-                     of those who navigate using speech recognition software.
+                  <p>最後に、<code>aria-describedby</code> はアクセシブルな名前 (accessible name) 計算には含まれない（代わりに、アクセシブルな説明の計算の一部となる）。慣例により、<code>aria-describedby</code> によってコントロールに関連付けられたテキストは、スクリーンリーダーによってアクセシブルな名前 (accessible name) の直後に通知される。したがって、見出し、指示、及びグループラベルのコンテキストは、音声認識ソフトウェアを用いてナビゲートする利用者のエクスペリエンスに影響を与えることなく、スクリーンリーダーの利用者を支援するために、アクセシブルな説明を通して提供される。
```

Comment:
「の」抜けかも

エクスペリエンスは体験で良いような?

```suggestion
                  <p>最後に、<code>aria-describedby</code> はアクセシブルな名前 (accessible name) の計算には含まれない（代わりに、アクセシブルな説明の計算の一部となる）。慣例により、<code>aria-describedby</code> によってコントロールに関連付けられたテキストは、スクリーンリーダーによってアクセシブルな名前 (accessible name) の直後に通知される。したがって、見出し、指示、及びグループラベルのコンテキストは、音声認識ソフトウェアを用いてナビゲートする利用者の体験に影響を与えることなく、スクリーンリーダーの利用者を支援するために、アクセシブルな説明を通して提供される。
```